src/components/SQForm/SQFormInclusionListItem.tsx

```diff
@@ -1,19 +1,33 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import Checkbox from '@material-ui/core/Checkbox';
 import FormControlLabel from '@material-ui/core/FormControlLabel';
 import Grid from '@material-ui/core/Grid';
 
 import {useForm} from './useForm';
 
+interface SQFormInclusionListItemProps {
+  /** evaluation of whether the box should be checked */
+  isChecked: boolean;
+  /** Disabled state of the checkbox */
+  isDisabled?: boolean;
+  /** Descriptive label text for the checkbox */
+  label: string;
+  /** name of the checkbox array in `initialValues` */
+  name: string;
+  /** Custom onChange event callback */
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  /** Size of the input given full-width is 12. */
+  size?: 'auto' | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
```

Comment:
There's a `GridSize` type provided in MUI - you can see it used in `SQFormCheckboxGroup.tsx` - credit to @20BBrown14 for showing me this a couple of weeks ago.

Reply:
You can also extend `BaseFieldProps`, then you won't need to specify the types for `name`, `label`, and `size`
```ts
interface SQFormInclusionListItemProps extends BaseFieldProps {
  /** evaluation of whether the box should be checked */
  isChecked: boolean;
  /** Disabled state of the checkbox */
  isDisabled?: boolean;
  /** Custom onChange event callback */
  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
}
```

Reply:
Great suggestions guys. Thanks for the feedback. Making those changes.